README.md

```diff
@@ -37,6 +37,10 @@ libpcap should be installed on Mac OS X by default.
 
 **Note:** A timeout of zero may cause ```pcap::Capture::next``` to hang and never return (because it waits for the timeout to expire before returning). This can be fixed by using a non-zero timeout (as the libpcap manual recommends) and calling ```pcap::Capture::next``` in a loop.
 
+## Specifying library folder
+
```

Comment:
For now I'm happy keeping it where it is, but if it became a larger project it would be worth putting it in some kind of org. (With multiple repos.)

Adding you as a contributor.